css/index.css

```diff
@@ -1,3 +1,420 @@
+}
+
+.services-list{
+	display: flex;
+	flex-direction: column;
+}
+
+.services-row {
+	display:flex;
+	flex-direction: row;
+}
+
+.services-box{
+	display: flex;
+	flex-direction: column;
+	align-content: space-around;
+	align-items: flex-start;
+	padding: 20px;
+	border: 1px solid black;
+	margin: 20px;
+	background-color:#e6ebed;
+}
+
+.learn-more-button{
+	padding: 10px;
+	margin-top: 15px;
+	text-align: center;
+	border: 1px solid black;
+	border-radius: 5px;
+	background-color: white;
+}
+
+.services-facts{
+	display: flex;
+	flex-direction: row;
+}
+
+.services-facts-text{
+	display: flex;
+	flex-direction: column;
+	align-content: space-around;
+	padding: 20px;
+	margin: 20px;
+}
+
+.services-info-img {
+	padding: 20px;
+}
+
+.services-facts-text-component{
+	padding: 5px;
+}
+
+
+/*Media Queries*/
+
+/*Mobile*/
+@media (max-width:500px) {
+
+    body {
+        border: 1px solid gray;
+    }
+    .container {
+        max-width: 90%;
+    }
+
+    header {
+        width:100%;
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: space-evenly;
+        flex-wrap: wrap;
+        align-items: center;
+    }
+
+    header nav {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        align-items: center;
+        flex-shrink: 1;
+        width: 100%;
+        margin-top: 35px;
+    }
+
+    header .logo {
+        width:60%;
+    }
+
+    header nav a {
+        display: flex;
+        align-items: center;
+        width: 100%;
+        border-bottom: 1px solid gray;
+        border-bottom-width: 100%;
+
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .cta {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        flex-wrap: wrap;
+        align-items: center;
+    }
+
+    .main-content .top-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        align-content: center;
+        padding: 0 10px;
+    }
+
+    .main-content .top-content .text-content {
+        margin-top: 10px;
+        margin-bottom: 10px;
+        width: 100%;
+    }
+
+    .main-content .middle-img {
+        height: 100px;
+        margin-top: 45px;
+      }
+      
+    .main-content .bottom-content {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-evenly;
+        margin-top: 37px;
+        padding: 0 10px;
+    }
+}
+
+/*Tablet*/
+@media (min-width: 501px) and (max-width: 800px) {
+    
+    .container {
+        max-width: 90%;
+    }
+
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: space-evenly;
+        flex-wrap: wrap;
+        align-items: center;
+    }
+
+    header nav {
+        display: flex;
+        justify-content: space-evenly;
+        flex-shrink: 1;
+        width: 100%;
+        margin-top: 35px;
+    }
+
+    header nav a {
+
+    }
+
```

Comment:
The logo is a bit small in tablet view. You can use the `width` property to make it larger.